lib/galaxy/datatypes/gis.py

```diff
@@ -2,7 +2,11 @@
 GIS classes
 """
 
+import nice_size
```

Comment:
This import is not correct. You should have:

`from galaxy.util import nice_size`

Reply:
Thank you Anne!